zerver/webhooks/stripe/view.py

```diff
@@ -53,6 +55,14 @@ def api_stripe_webhook(
     payload: Dict[str, Any] = REQ(argument_type="body"),
     stream: str = REQ(default="test"),
 ) -> HttpResponse:
+    sig_header = request.META["HTTP_STRIPE_SIGNATURE"]
```

Comment:
Can you add a comment linking to the Stripe documentation for their signature feature?

Reply:
i have referred this document https://stripe.com/docs/webhooks/signatures

Reply:
@madrix01 We should add that link in a comment in the code so that someone reading the code can visit the link, if need be. :)

Reply:
oh sorry my bad, i'll add the link to the code